content/new-in-22.md

```diff
@@ -199,40 +189,51 @@ Provide ways to help users navigate, find content, and determine where they are.
 </blockquote>
 <p><a href="https://www.w3.org/WAI/WCAG22/Understanding/focus-not-obscured-minimum">Understanding Focus Not Obscured (Minimum)</a></p>
 
-### 2.4.12 Focus Not Obscured (Enhanced) (AAA)
-<p><em>(Persona, problem, and works well<!--, and "in brief" --> same as 2.4.12 above.)</em></p>
+### [EOWG draft] 2.4.12 Focus Not Obscured (Enhanced) (AAA)
+
+<p>In brief:</p>
+<ul>
+      <li><p><span class="whatwhy">What to do: </span>Ensure that when items have focus, they are fully visible.</p></li>
+      <li><p><span class="whatwhy">Why do it: </span>Some people cannot use a mouse and rely on seeing what has keyboard focus.</p> </li>
+</ul>
+<p><em>(Persona, problem, and works well same as 2.4.12 above.)</em></p>
 <p class="sclabel">WCAG:</p>
 <blockquote class="sc">
   <p>When a <a href="https://www.w3.org/TR/WCAG22/#dfn-user-interface-components">user interface component</a> receives keyboard focus, no part of the component is hidden by author-created content.</p>
 </blockquote>
 <p><a href="https://www.w3.org/WAI/WCAG22/Understanding/focus-not-obscured-enhanced">Understanding Focus Not Obscured (Enhanced)</a></p>
 
-### 2.4.13 Focus Appearance (AAA)
-<p class="persona"><a href="https://www.w3.org/WAI/people-use-web/user-stories/#reporter">Reporter</a> with repetitive stress injury who doesn't use a mouse:<br>
-  and <a href="https://www.w3.org/WAI/people-use-web/user-stories/#retiree">Retiree</a> with low contrast sensitivity:</p>
-<div class="quotes">
-  <ul>
-    <li>
-      <p><span class="issue">Problem:</span><span><q>I can't tell where the keyboard focus is as I move around a web page or app.</q></span></p>
-    </li>
-    <li>
-      <p><span class="issue">Works well:</span><span><q>I can see where the keyboard focus is as I move around a web page or app.</q></span></p>
-    </li>
-  </ul>
-</div>
-<!--
+### [AG version] 2.4.13 Focus Appearance (AAA)
 <section class="brief">
   <p>In brief:</p>
   <dl>
     <dt>Objective</dt>
     <dd>Make it easier to see what has keyboard focus</dd>
     <dt>Author task</dt>
-    <dd>The focus indicator contrasts 3:1 with both the component’s unfocused state and its surroundings</dd>
+    <dd>The focus indicator color contrasts 3:1 with both the component’s unfocused state and its surroundings</dd>
     <dt>Key beneficiaries</dt>
     <dd>Sighted users reliant on keyboard interaction</dd>
   </dl>
 </section>
--->
+
+### [EOWG draft] 2.4.13 Focus Appearance (AAA)
+<p>In brief:</p>
+<ul>
+      <li><p><span class="whatwhy">What to do: </span>Ensure that the focus indicator is at least 2px and contrasts 3:1 with the component’s unfocused state and with its surroundings.</p></li>
+      <li><p><span class="whatwhy">Why users need it: </span>People with low contrast sensitivity cannot see small contrast differences.</p></li>
```

Comment:
I'm finding the variability of the prefixes are making it harder to consistently assess these. For example, these first three have variations:
- why
- why do it
- why users need it

Obviously those lead to different wording (e.g., I'm not going to say "need" again if the prefix says "need").

I'm going to first incorporate the Why, since for the most part it is relatively straight forward to translate the currently adopted "Author task" to meet this (regardless of the prefix variation).

If EOWG can settle on their preference for the prefix, it'll help.
"What to do" seems to work fine to me (and is likely more meaningful to folks than 'author task')

There is undoubtedly a different timber to a prefix called "objective", or better, "goal", than one called "why". 
Having written those goals, I still would like to ask for the group's re-consideration on the value of a simple message like "Make it easier to spot the keyboard focus" or "Make controls easier to activate".

Reply:
> I'm finding the variability of the prefixes are making it harder to consistently assess these.

Sure. They are there so EOWG could see the different options. It's out for [survey](https://www.w3.org/2002/09/wbs/35532/wcag2_2_in_brief/?login) now -- the last question. That survey is open to anyone, so feel free to add your input. (We moved the closing date of that survey to Tuesday, which is a tighter timeline than EOWG is used to, so not sure how many replies we'll get.)

> I'm going to first incorporate the Why

I agree with some of your edit suggestions, though not all. Probably better that we resolve things here before you put it in WCAG space so we can discuss in one place.

Would you be willing to get on the phone to discuss these Tuesday?

Reply:
> Having written those goals, I still would like to ask for the group's re-consideration on the value of a simple message like "Make it easier to spot the keyboard focus" or "Make controls easier to activate".

Do you mean in addition to the what and why? So:
* Goal:
* What to do:
* Why:

or instead of Why?

Depending on which, I'll have additional input. :-)

<hr>

Though I will say now that "easier" seems not to convey the importance for some users.

Reply:
Suggesting instead of. I'm just musing.
I'm also exploring the mashup of the goal with the target users, as suggested somewhere in the conversation.
I'm going to let my mind percolate on this over part of the weekend.

Reply:
> I agree with some of your edit suggestions, though not all. Probably better that we resolve things here before you put it in WCAG space so we can discuss in one place.

@shawna-slh I have not made any changes in the WCAG In Briefs based on our iterations in this file; however, some of the Author task phrases were updated in WCAG already based on user feedback, so some of the changes I've made to 'what to do' are based on that existing WCAG wording.

Reply:
Based on EOWG input -- and the fact that many people reading the Understanding docs don't understand the needs of people with disabilities -- going with "Why it's important" for now... to see how that fits.